### PR TITLE
Added arm64 jobs in Travis-CI

### PR DESCRIPTION
**Pull request by ossdev07**
Travis-CI has added support for ARM64. Added ARM64 jobs in Travis-CI.

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -223,7 +223,7 @@ def is_stable(extra):
 htmlhelp_basename = 'ipythondoc'
 
 intersphinx_mapping = {'python': ('https://docs.python.org/3/', None),
-                       'rpy2': ('https://rpy2.readthedocs.io/en/version_2.8.x/', None),
+                       'rpy2': ('https://rpy2.readthedocs.io/en/latest/', None),
```
Comment (rhenwood-arm) [inline docs/source/conf.py]: You should explain in the commit message why you have updated this line, as it is not obvious why it relates to adding an Arm64 Travis-ci job.

Comment (Carreau) [reply docs/source/conf.py]: That has been fixed in master; I'll rebase so that it does not appear in the PR

**Comment by Carreau** [comment]
Oh, that's a lot of new entry in the test-matrix. Do you think they are all necessary ? It will likely quite increase the time to run all the tests. We also do not have compiled code; so is ti really worth it ; or should we maybe test only on things that would potentially break on CPython ie the -dev or 3.8 and nightly ?

**Comment by Carreau** [comment]
Merging for now; but if that make CI too slow; I might trim a few.

**Pull request merged**